src/tests/test_common/hs_repl_test_common.hpp

```diff
@@ -172,8 +172,7 @@ class HSReplTestHelper {
             name_ + std::to_string(replica_num_),
             {{HS_SERVICE::META, {.size_pct = 5.0}},
              {HS_SERVICE::REPLICATION, {.size_pct = 60.0, .repl_app = std::make_unique< TestReplApplication >(*this)}},
-             {HS_SERVICE::LOG_REPLICATED, {.size_pct = 20.0}},
-             {HS_SERVICE::LOG_LOCAL, {.size_pct = 2.0}}});
+             {HS_SERVICE::LOG, {.size_pct = 22.0}}});
```

Comment:
Nit: We can still retain 20% :-). That 2% was because we can't give 0%.

Reply:
Yeah :) , just trying to avoid any regression.